pkg/relay/ethereum/tx.go

```diff
@@ -278,3 +268,69 @@ func (c *Chain) TxAcknowledgement(opts *bind.TransactOpts, msg *chantypes.MsgAck
 		ProofHeight:     pbToHandlerHeight(msg.ProofHeight),
 	})
 }
+
+func (c *Chain) SendTx(opts *bind.TransactOpts, msg sdk.Msg, noSend bool) (*gethtypes.Transaction, error) {
+	logger := c.GetChainLogger()
+	opts.NoSend = noSend
```

Comment:
This `opts.NoSend = xxx` can be moved out of this function, right?

Reply:
Fixed